src/Sarif.Driver.UnitTests/SimpleTestRule.cs

```diff
@@ -12,7 +12,7 @@ namespace Microsoft.CodeAnalysis.Sarif
     [Export(typeof(IRule)), Export(typeof(IOptionsProvider)), Export(typeof(ISkimmer<TestAnalysisContext>))]
     internal class SimpleTestRule : TestRuleBase, IOptionsProvider
     {
-        public new string Id => "TEST002";
+        public override string Id => "TEST002";
```

Comment:
>override [](start = 15, length = 8)

oops, use of new compromised ability of this code to perform correctly when access through an IRule reference.